typescript/lib/services/container_creation_config.ts

```diff
@@ -0,0 +1,110 @@
+// package services
```

Comment:
No need to do it now, but we'll need to delete these before final merge into `develop`